? p/news/archives
? p/news/class.news.php
? p/news/en
? p/news/fr-body
? p/news/fr-title
? p/news/show
cvs diff: Diffing .
Index: style.css
===================================================================
RCS file: /var/cvs/e/web/www/style.css,v
retrieving revision 1.12
diff -u -r1.12 style.css
--- style.css	30 Apr 2007 04:09:51 -0000	1.12
+++ style.css	8 May 2007 09:47:22 -0000
@@ -389,6 +389,48 @@
 }
 
 
+/** NEWS STYLE */
+
+.post {
+   margin                 : 0 0 2.5em 40px;
+   border-left            : 1px solid #DDD;
+   padding-left           : 2ex;
+   position               : relative;
+}
+
+html>body .news-date{
+   left                   : -11ex;
+}
+
+.news-date {
+   margin                 : 0;
+   text-align             : right;
+   width                  : 10ex;
+   position               : absolute;
+   left                   : -13ex;
+   font-size              : 90%;
+   font-family            : "Bitstream Vera", "Vera", "Trebuchet MS", Trebuchet, Tahoma, sans-serif;
+   color                  : #888888;
+}
+
+.news-title {
+   font-size              : 1.1em;
+   margin-bottom          : -1.2ex;
+}
+
+.news-title a, .news-title a:visited, .news-title a:hover {
+   text-decoration        : none;
+}
+
+.news-title span {
+   font-size              : 0.7em;
+}
+
+.news-content {
+   text-align             : justify;
+}
+
+
 
 /* OLD CSS STUFF TO STEAL FROM */
 
cvs diff: Diffing i
cvs diff: Diffing p
cvs diff: Diffing p/about
cvs diff: Diffing p/about/community
cvs diff: Diffing p/about/dev
cvs diff: Diffing p/about/dev/d
cvs diff: Diffing p/about/e16
cvs diff: Diffing p/about/libs
cvs diff: Diffing p/about/libs/bindings
cvs diff: Diffing p/about/libs/ecore
cvs diff: Diffing p/about/libs/edbus
cvs diff: Diffing p/about/libs/edje
cvs diff: Diffing p/about/libs/eet
cvs diff: Diffing p/about/libs/efreet
cvs diff: Diffing p/about/libs/embryo
cvs diff: Diffing p/about/libs/emotion
cvs diff: Diffing p/about/libs/etk
cvs diff: Diffing p/about/libs/evas
cvs diff: Diffing p/about/libs/ewl
cvs diff: Diffing p/about/libs/imlib2
cvs diff: Diffing p/about/people
cvs diff: Diffing p/docs
cvs diff: Diffing p/docs/wiki
cvs diff: Diffing p/download
cvs diff: Diffing p/download/cvs
cvs diff: Diffing p/download/dev
cvs diff: Diffing p/download/e16
cvs diff: Diffing p/download/libs
cvs diff: Diffing p/index
cvs diff: Diffing p/index/d
cvs diff: Diffing p/main1
cvs diff: Diffing p/main2
cvs diff: Diffing p/main3
cvs diff: Diffing p/main4
cvs diff: Diffing p/main5
cvs diff: Diffing p/main6
cvs diff: Diffing p/main7
cvs diff: Diffing p/main8
cvs diff: Diffing p/news
Index: p/news/en-body
===================================================================
RCS file: /var/cvs/e/web/www/p/news/en-body,v
retrieving revision 1.3
diff -u -r1.3 en-body
--- p/news/en-body	22 Apr 2007 04:21:16 -0000	1.3
+++ p/news/en-body	8 May 2007 09:47:22 -0000
@@ -1,9 +1,28 @@
 <h1>News</h1>
-<p>
-Now web.enlightenment.org should auto-update within 1 minute of any CVS commit
-to the www module. I am making this change here to test to see if that actually
-works now.
-</p><p>
-*FIXME* - this needs to become site php to show all items from the news
-repository with the most recent 10 here and older ones linked to.
-</p>
+
+<?php
+require_once 'class.news.php';
+
+setlocale(LC_ALL, 'en_EN');
+
+$news = new news;
+$list = $news->get(10);
+foreach($list as $new)
+{
+   if (($new = $news->read_content($new)) !== NULL)
+   {
+     echo "<div class='post'>\n";
+     echo "<p class='news-date'>".$new['date']."</p>\n";
+     echo "<p class='news-title'><a href='p.php?p=news/show&l=".$_GET['l']."&news_id=$new[id]'>".$new['title']."</a></p>";
+     echo "<p class='tiny'><a href='mailto:".$new['email']."'>".$new['author']."</a> - ".$new['date']."</p>";
+     echo "<p class='news-content'>".$new['content']."</p>\n";
+     echo "</div>\n";
+   }
+}
+
+if ($news->has_archives() == true)
+{
+   echo "<p align='right'>more in the <a href='p.php?p=news/archives&l=".$_GET['l']."' title='Read archives'>archive</a></p>";
+}
+
+?>
cvs diff: Diffing p/sponsors
cvs diff: Diffing site
